installing/installing_alibaba/installing-alibaba-customizations.adoc

```diff
@@ -0,0 +1,65 @@
+:_content-type: ASSEMBLY
+[id="installing-alibaba-customizations"]
+= Installing a cluster on Alibaba Cloud with customizations
+include::modules/common-attributes.adoc[]
+:context: installing-alibaba-customizations
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a customized cluster on infrastructure that the installation program provisions on Alibaba Cloud. To customize the installation, you modify parameters in the `install-config.yaml` file before you install the cluster.
+
+[NOTE]
+====
+The scope of the {product-title} installation configurations is intentionally narrow. It is designed for simplicity and ensured success. You can complete many more {product-title} configuration tasks after an installation completes.
+====
+
+:FeatureName: Alibaba Cloud on {product-title}
+include::snippets/technology-preview.adoc[]
+
+== Prerequisites
```

Comment:
[id="prerequisites_installing-alibaba-customizations"]
per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies